tools/packaging/static-build/ovmf/build-ovmf.sh

```diff
@@ -79,9 +79,14 @@ popd
 
 info "Install fd to destdir"
 install_dir="${DESTDIR}/${PREFIX}/share/ovmf"
-if [ "${ovmf_build}" == "tdx" ]; then
-	install_dir="$DESTDIR/$PREFIX/share/tdvf"
-fi
+case "${ovmf_build}" in
+	"tdx")
+		install_dir="$DESTDIR/$PREFIX/share/tdvf"
```

Comment:
Is there a reason we are putting the firmware files in `share` rather than in `bin`?  @fidencio

Reply:
qemu traditionally keeps OVMF/firmware files in `/usr/share/...`